docs/Advanced.md

```diff
@@ -209,3 +209,21 @@ The documentation was moved to [Appfile.md](https://github.com/KrauseFx/fastlane
 ## Skip update check when launching `fastlane`
 
 You can set the environment variable `FASTLANE_SKIP_UPDATE_CHECK` to skip the update check.
+
+## Gitignore
+
+If you are using Git for this project, we recommend that you keep the `fastlane` configuration files in your repository. You may want to add the following lines to your `.gitignore` file to exclude some generated and temporary files:
+
+    # Fastlane temporary profiling data
+    /fastlane/report.xml
+    # Deliver temporary error output
+    /fastlane/Error*.png
+    # Deliver temporary preview output
+    /fastlane/Preview.html
+    # Snapshot generated screenshots
+    /fastlane/screenshots/*/*-portrait.png
+    /fastlane/screenshots/*/*-landscape.png
+    /fastlane/screenshots/screenshots.html
+    # Frameit generated screenshots
+    /fastlane/screenshots/*/*-portrait_framed.png
+    /fastlane/screenshots/*/*-landscape_framed.png
```

Comment:
Not sure what you mean, you can just copy them from here https://github.com/fastlane/fastlane/blob/bfc7a6e277bbede8b15c1d89a31ebdd7c5e90b63/docs/Advanced.md#gitignore